spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationAutoConfiguration.java

```diff
@@ -131,4 +143,47 @@ public IntegrationMBeanExporter integrationMbeanExporter() {
 
 	}
 
+	/**
+	 * Integration JDBC configuration.
+	 */
+	@Configuration
+	@ConditionalOnClass(JdbcMessageStore.class)
```

Comment:
Well, I deliberately left it this way - `@ConditionalOnClass(JdbcMessageStore.class)` is used to detect the presence of `spring-integration-jdbc` and is applied to `IntegrationJdbcConfiguration` class whereas `@Conditional(IntegrationSchemaCondition.class)` is only applied to initializer bean.

This way we can add more JDBC related stuff to `IntegrationJdbcConfiguration` in the future (which might not need schema initializer) without changing conditions.